CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Add an experimental key for internal team use:
   `r2c-internal-project-depends-on` that allows rules to filter based on the
   presence of 3rd-party dependencies at specific version ranges.
+- FOR INTERNAL USE: if the environment variable SEMGREP_R2C_INTERNAL_EXPLICIT_SEMGREPIGNORE
```

Comment:
My bad, I can make a PR removing it from the changelog, or you can just delete it when you do the release. The utility of the feature, which I mention in the PR description but probably in not enough detail, is that semgrep-action needs to read an extra set of ignore rules from semgrep-app, and append these to whatever is being ignored by a user's .semgrepignore file. But we don't actually want to edit a user's .semgrepignore file, since that's likely to cause confusion. They run semgrep-action and suddenly new stuff is getting ignored by the CLI. So this lets semgrep-action create a temporary file of ignore rules, without editing or renaming the user's .semgrepignore, and force the CLI to follow the new temporary file.